joomla-projects/joomla-cms
# Create a Last Visit Date filter on the user list

#### Summary of Changes
The user list has a number of filters, one of them is the registration date. However I think that the last visit date is more useful as it can show when a user last came to the site. If that is too long ago the user account may be considered obsolete. So this PR adds a filter for the last visit date.

#### Testing Instructions
1. Go to Users -> Manage
2. Click on Search Tools, you will see these filters:
![image](https://cloud.githubusercontent.com/assets/359377/14934726/c5be4350-0eba-11e6-9715-f6d5ad97d96f.png)
3. Apply the patch
4. Go to Users -> Manage
5. Click on Search Tools, you will see these filters:
![image](https://cloud.githubusercontent.com/assets/359377/14934733/f442e5f0-0eba-11e6-8b10-7ebca85976ec.png)
6. You now have a filter for the Last Visit Date
7. Check if the filter works as expected.
